### PR TITLE
Update br.m3u

### LINKED ISSUE #2842: Add: Loading
**Channel Name:** Loading
**Country:** Brazil
**Language:** Portuguese
**External Link (optional):** https://akamai.gcp.br.cdn.booyah.live/hls/151073/34850987_1080.m3u8
**Notes (optional):** tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/2/2e/Loading_TV.svg/640px-Loading_TV.svg.png" group-title="Entertainment"

### PR DESCRIPTION
**Pull request by freearhey**
Resolves #2842

Added links from https://github.com/iptv-org/iptv/issues/1968#issuecomment-846496931 and https://github.com/iptv-org/iptv/issues/1968#issuecomment-846703161

### CODE REVIEW

**Comment by LaneSh4d0w** [comment]
Will take a look soon 👍

File: channels/br.m3u [hunk 1]
```diff
@@ -315,3 +315,33 @@ http://unitvaovivo.ufrgs.br:8080/live.ogg
 https://srv5.zcast.com.br/andre4369/andre4369/playlist.m3u8
 #EXTINF:-1 tvg-id="Yeeaah.br" tvg-name="Yeeaah!" tvg-country="BR" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/TxdOyKS.png" group-title="General",Yeeaah! (720p)
 https://srv3.zcast.com.br/yeeaah/yeeaah/playlist.m3u8
+#EXTINF:-1 tvg-id="Loading.br" tvg-name="Loading" tvg-country="BR" tvg-language="Portuguese" tvg-logo="" group-title="",Loading (1080p) [Geo-blocked]
+https://google.gcp.br.cdn.booyah.live/hls/151073/master_34850987.m3u8
```
Comment (LaneSh4d0w) [inline channels/br.m3u:319]: Geo-blocked ? Works for me.
vlcsnap-2021-05-31-17h41m37s334

File: channels/br.m3u [hunk 2]
```diff
@@ -315,3 +315,33 @@ http://unitvaovivo.ufrgs.br:8080/live.ogg
...
+#EXTINF:-1 tvg-language="Portuguese" tvg-logo="https://www.upf.br/Assets/img/logo.png" group-title="Science",TV UFOP
+https://edge-df-01.eduplay.rnp.br/media/ocp(r(iRGrcw)t(AAAAAAAAAAA)p(d(Dxg)k(Qdk)m(6pbmcjLXNoerDLdAhoXBxw)n(a(3LDBWxJLACwHVOavx5cO4BC5lA2ZDpPC)s(HrY)'a(12ztgSdgu3rQqz6xH52ZjVZa)s(Pm8)))s(t(aTo)b(PTsuEXhXAScNjp6w9ssG70foQQQiZk9_8m0vsjgLJ_yzxGPuj8i57PEDMMA4cmTNsKda2B5oX-EEErouZ1HFyuXyAosvuqvlMJtsMkooySBOlQbVM33PgsYPQdea2CdwV3jCwh3bEWxgkPO8qmBPQtt_5bUEV1Mi_2t1AjM)'t(S2o)b(WaBWYarWoWCInKe38WSze3Q_-kcUr5z4MZQ)))/live_720_index.m3u8
+#EXTINF:-1 tvg-language="Portuguese" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/b/b9/UFOP_logo.gif" group-title="Science",TV UFOP
+https://edge-df-01.eduplay.rnp.br/media/ocp(r(iRGrcw)t(AAAAAAAAAAA)p(d(Dxg)k(Qdk)m(6pbmcjLXNoerDLdAhoXBxw)n(a(3LDBWxJLACwHVOavx5cO4BC5lA2ZDpPC)s(HrY)'a(12ztgSdgu3rQqz6xH52ZjVZa)s(Pm8)))s(t(aTo)b(PTsuEXhXAScNjp6w9ssG70foQQQiZk9_8m0vsjgLJ_yzxGPuj8i57PEDMMA4cmTNsKda2B5oX-EEErouZ1HFyuXyAosvuqvlMJtsMkooySBOlQbVM33PgsYPQdea2CdwV3jCwh3bEWxgkPO8qmBPQtt_5bUEV1Mi_2t1AjM)'t(S2o)b(WaBWYarWoWCInKe38WSze3Q_-kcUr5z4MZQ)))/live_720_index.m3u8
```
Comment (LaneSh4d0w) [inline channels/br.m3u:329]: Duplicate.

**Review by BellezaEmporium — CHANGES_REQUESTED** [review]

File: channels/br.m3u (hunk 1 above)
Comment (freearhey) [reply channels/br.m3u:319]: Yeah, you're right. Apparently only the player was blocked.

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**